test/studies/adventOfCode/2022/day03/bradc/day03.chpl

```diff
@@ -0,0 +1,54 @@
+// Concepts for Blog:
+// - bytes type
+//   - .size, slicing, iteration, toByte, literals
+// - forall loops
+//   - reduce intents
+// - sets
+//   - add, contains
+
+use IO, Set;
+
+var Rucksacks = readItems();
+writeln(sumOfPriorities(Rucksacks));
+
+iter readItems() {
+  var rucksack: bytes;
+  while readLine(rucksack) {
+    const len = rucksack.size-1,
+          mid = len / 2;
+    yield (rucksack[0..<mid], rucksack[mid..<len]);
```

Comment:
We could use unbounded ranges here, right?

Reply:
Because I didn't drop the linefeed from the string (because I think of strip() as being expensive and this week's bug has made me paranoid about using the optional argument to readLine()), I can't use it for the second slice.  And because I wasn't using it for the second slide, I didn't want to use it for the first to avoid asymmetry.  Also it meant one less thing for me to explain on the weekend.  :D  

But also, I can never recall whether we support open-interval upper-unbounded ranges (my guess is "not currently", and I think Jeremiah and I have expressed wishes for them at times.